packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -150,9 +150,10 @@ class DeepLinksController extends DisposableController {
   String get applicationId =>
       _androidAppLinks[selectedVariantIndex.value]?.applicationId ?? '';
 
+  @visibleForTesting
   List<LinkData> get getLinkDatasByPath {
```

Comment:
not part of this PR, but we should avoid starting method names / getters with "get". Can you rename all methods / getters that violate that best practice?
https://dart.dev/effective-dart/design#avoid-starting-a-method-name-with-get